translate/translate.py

```diff
@@ -33,11 +34,11 @@ def main():
     if lang not in ('ko', 'en'):
         raise ValueError('Unknown language')
     data = translator.translate(text, dest=LANG[lang]).text
-    pyperclip.copy(data)
+    if args.clipboard:
```

Comment:
This removes the ability for the translation to be automatically saved to the _clipboard_. The `args.clipboard` option has a different meaning.

Reply:
this part is rolled back.
fed402e5504c408c1f83c0fff174f07fdbb3d98b